package.json

```diff
@@ -9,12 +9,13 @@
     "@testing-library/react": "^9.4.0",
     "@testing-library/user-event": "^7.2.1",
     "i18next": "^19.1.0",
+    "muicss": "^0.10.1",
```

Comment:
What is mui css? Do we need it?

Reply:
From when I was researching how to get a drop down inside the tab, it was the only component I could find that was able to fit in the tab but is still a bit buggy